backend/tests/test_auth.py

```
@@ -334,3 +337,151 @@ async def test_login_google_only_user_with_password(
+        mock_user = Mock()
+        mock_user.id = "user_123"
+        mock_user.email = "test@example.com"
+        mock_link.return_value = mock_user
+
+        # Mock the access token
+        mock_create_token.return_value = "mock_access_token"
+
+        google_login_data = {"code": "valid_google_code"}
+        response = await async_client.post("/api/auth/google", json=google_login_data)
+
+        assert response.status_code == 200
```

Ruff (0.12.2)

Use of `assert` detected

(S101)